### ISSUE TITLE
[Legitimate Site Blocked] tezos.finance

### ISSUE THREAD

**Issue by KevinMehrabi** [MetaMask/eth-phishing-detect#9692]
https://tezos.finance/ is a legitimate website application from reputable developers. The application is by [StableTech](https://stable.tech) and VC-backed by [Draper Goren Holm](https://drapergorenholm.com). The website application does not even facilitate Metamask wallets, nor any EVM wallet for that matter. It is strictly on the Tezos network.

Please unblock and verify this website.

Our social media is:
[https://twitter.com/tezosfinance](https://twitter.com/tezosfinance)
[https://discord.gg/tezosfinance](https://discord.gg/tezosfinance)
[https://medium.com/tezosfinance](https://medium.com/tezosfinance)
[https://telegram.com/tezosfinance](https://telegram.com/tezosfinance)

**Comment by 409H** [comment]
Closing. Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/9693
